.github/workflows/helm-build-push.yml

```diff
@@ -0,0 +1,53 @@
+name: Helm chart release on tag
+
+on:
+  workflow_call:
+    charts:
+      type: str
+      description: "What Charts should be updated? Can be comma separated list."
+
+jobs:
+  release-helm-chart:
+    name: Release helm chart
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        chart:
+          [ split(${{ github.event.inputs.charts }}, ',') ]
```

Comment:
I am looking forward if this works at the end :-)

Reply:
Me too :D